src/consensus/aft/raft_types.h

```diff
@@ -141,6 +158,14 @@ namespace aft
     bool success;
   };
 
+  struct SignedAppendEntriesResponse : RaftHeader
+  {
+    Term term;
+    Index last_log_idx;
+    uint32_t signature_size;
```

Comment:
Why does the size of the signature need to be passed in if the signature is an `std::array`?

Reply:
ECDSA signatures are variable sized. The macro MBEDTLS_ECDSA_MAX_LEN defines the maximum possible length and signature_size is the size of the actual signature. The max size is 141 bytes so its no actually that big

Reply:
I think Julien's point was that std::array already has a size()?